src/components/theme-switch.component.astro

```diff
@@ -0,0 +1,66 @@
+---
+import LightIcon from './icons/light.icon.astro';
+import DarkIcon from './icons/dark.icon.astro';
+---
+
+<style lang="scss">
+  .theme-switch {
+    &.light,
+    &.dark {
+      cursor: pointer;
+      display: none;
+
+      &.show {
+        display: block;
+      }
+    }
+  }
+</style>
+
+<div theme-switch>
```

Comment:
As we do not have a custom element, this attribute won't be valid. Either use the `data-` prefix, or use an id / className.